website/src/css/_gradients.scss

```diff
@@ -9,14 +10,28 @@
 @mixin BgGradientBlueGreen {
     background: linear-gradient(90deg, $light-blue 0%, $light-green 100%);
 }
+@mixin BgGradientOnahauMint {
+    background: linear-gradient(
+        90deg,
+        color.adjust($brand-onahau, $alpha: -0.7) 14.33%,
+        color.adjust($brand-mint, $alpha: -0.7) 81.79%
+    );
+}
+@mixin BgGradientOnahauFog {
```

Comment:
Ohh so happy you are adding these! 

QQ for you:  #5172 uses a version of `bg-gradient-onahau-fog`
![Screen Shot 2022-03-15 at 10 56 21 AM](https://user-images.githubusercontent.com/43454804/158406192-8d4fa5cd-ee9e-443d-8f01-f65caa0107ec.png)

My vote would be to use the same background on both of these, rather than creating a new background just to reverse directions. What do you think? I could drop this `bg-gradient-onahau-fog` into my PR, and then we would be consistent between pages.

Reply:
Yeah that sounds good to me! Let's use the same background!